### ISSUE TITLE
Online Javadoc search produces broken links containing 'undefined/' string

### ISSUE THREAD

**Issue by szarnyasg** [jgrapht/jgrapht#884]
```
 * JGraphT version: 1.4.0
 * Java version (java -version)/platform: online Javadoc
```

**Issue**

When using the "Search" box in the top right corner of the Javadoc, it leads to links with `undefined/` in them, e.g. <https://jgrapht.org/javadoc/undefined/org/jgrapht/alg/shortestpath/BFSShortestPath.html> (this results in a 404 error).

**Steps to reproduce (small coding example)**

1. Go to <https://jgrapht.org/javadoc/>.
2. Type `BFS` in the search box.
3. Click the first result, `org.jgrapht.alg.shortestpath.BFSShortestPath`.
4. A 404 page appears.

**Expected behaviour**

It should load the documentation site. A potential workaround now is to delete the `undefined/`  string from the URL which leads to the working site, e.g. <https://jgrapht.org/javadoc/undefined/org/jgrapht/alg/shortestpath/BFSShortestPath.html>.

**Comment by jsichi** [comment]
Appears to be due to this:

https://stackoverflow.com/questions/52326318/maven-javadoc-search-redirects-to-undefined-url

Maybe someone can submit a PR based on https://stackoverflow.com/a/56641766 ?

**Comment by jsichi** [comment]
This fixed itself as part of our 1.5.0 release, probably from one of our dependency upgrades.
